packages/cli/helpers/run-tests.ts

```diff
@@ -23,7 +23,16 @@ export async function main() {
     process.env.NODE_EXTRA_CA_CERTS = miniProxy.defaultCertificatesConfig.caCert
   }
 
-  execa.sync('jest', ['--silent', ...process.argv.slice(2)], {
+  // We pass the --passWithNoTests flag to jest and vitest for them not to fail
+  // when the filter provided via CLI only matches tests in one of the two suites.
```

Comment:
It might be better to use a more exclusive file filter, e.g. `subcommand.vitest.ts`

Reply:
We'd need this flag regardless, because it's exactly when only vitest or only jest matches, the other would fail by default.

But I like the idea of using `.vitest.ts` suffix for tests that should be running under Vitest instead of harcoding the specific file names in `jest.config.js` and `vitest.config.ts` 👍